Sources_v3/Controllers/ChannelController.swift

```diff
@@ -59,12 +59,7 @@ public typealias ChannelController = ChannelControllerGeneric<DefaultDataTypes>
 ///
 public class ChannelControllerGeneric<ExtraData: ExtraDataTypes>: Controller, DelegateCallable {
     /// The ChannelQuery this controller observes.
-    public private(set) var channelQuery: ChannelQuery<ExtraData> {
-        didSet {
-            guard oldValue.cid != channelQuery.cid else { return }
-            startDBObserving()
-        }
-    }
+    public private(set) var channelQuery: ChannelQuery<ExtraData>
```

Comment:
Let's make this `@Atomic` since it's public and we can also modify locally from a background thread.